src/tests/cart/S19_add_to_cart.test.js

```diff
@@ -1,41 +1,81 @@
+// tags: smoke, load, soak
 import { group } from 'k6';
 import OptionsUtil from '../../utils/options.util';
 import { createMetrics } from '../../utils/metric.util';
-import EnvironmentUtil from '../../utils/environment.util';
 import CartPage from '../../pages/yves/cart.page';
-import { ProductFixture } from '../../fixtures/product.fixture';
-
-const testConfiguration = {
-  ...EnvironmentUtil.getDefaultTestConfiguration(),
-  id: 'S19',
-  group: 'Cart',
-  metrics: ['S19_post_cart_add'],
-  thresholds: {
-    S19_post_cart_add: {
-      smoke: ['avg<300'],
-      load: ['avg<600'],
+import FixturesResolver from '../../utils/fixtures-resolver.util';
+import IteratorUtil from '../../utils/iterator.util';
+import exec from 'k6/execution';
+import ConfigResolver from '../../utils/config-resolver.util';
+import EnvironmentUtil from '../../utils/environment.util';
+import { LoginPage } from '../../pages/yves/login.page';
+import ProductPage from '../../pages/yves/product.page';
+import { parseHTML } from 'k6/html';
+import { sleep } from 'k6';
+
+const testConfiguration = new ConfigResolver({
+  params: {
+    id: 'S19',
+    group: 'Cart',
+    metrics: ['S19_post_cart_add'],
+    thresholds: {
+      S19_post_cart_add: {
+        smoke: ['avg<300'],
+        load: ['avg<600'],
+      },
     },
   },
-};
+}).resolveConfig();
```

Comment:
all test configurations should be same (structure), please avoid this new structure (we increasing complexity)